src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-rds-monitoring-integration.mdx

```diff
@@ -42,7 +42,7 @@ Default [polling](/docs/infrastructure/amazon-integrations/aws-integrations-list
 
 ## Find and use data [#find-data]
 
-To find your integration data, go to <DNT>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Infrastructure > AWS**</DNT> and select one of the RDS integration links.
+To view the RDS Dashboard, go to <DNT>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Infrastructure > AWS**</DNT> and scroll-down to RDS then select the RDS Dashboard link.
```

Comment:
There are too many steps in this sentence - it reads like a runon sentence. I'd recommend breaking it up into two sentences:
```suggestion
To view the RDS Dashboard:
1. Go to <DNT>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Infrastructure > AWS**</DNT>.
2. Scroll-down to <DNT>**RDS**</DNT> , then click <DNT>**RDS Dashboard**</DNT>.
```

Also, clarify what they're scrolling down on - the left-hand menu bar, for example